tests/ludwig/utils/test_config_utils.py

```diff
@@ -1,50 +1,50 @@
-from typing import Any, Dict, Optional
-
-import pytest
-
-from ludwig.utils.config_utils import merge_fixed_preprocessing_params
-
-
```

Comment:
Should we just delete this test module? I'm assuming you've commented out these tests because these functions are only used within the ModelConfig object, and they seem to work?

Reply:
I want to revisit in a follow-up to make it work with the new structure.